core/src/main/java/org/elasticsearch/http/netty/NettyHttpServerTransport.java

```diff
@@ -498,7 +509,10 @@ protected void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) thro
                 // ignore
                 return;
             }
-            if (!NetworkExceptionHelper.isCloseConnectionException(e.getCause())) {
+            if (e.getCause() instanceof CircuitBreakingException) {
```

Comment:
why do we need special handling for CircuitBreakingException ? REQUEST_ENTITY_TOO_LARGE doesn't feel right because it's not about this single request but rather about the accumulation  of all requests?